src/istio/control/http/attributes_builder_test.cc

```diff
@@ -765,6 +780,11 @@ TEST(AttributesBuilderTest, TestReportAttributes) {
         map["server"] = "my-server";
         return map;
       }));
+  EXPECT_CALL(mock_data, GetTracingHeaders(_))
+      .WillOnce(Invoke([](std::map<std::string, std::string> &m) {
```

Comment:
This is testing that if getTracingHeaders works correctly, the correct attribute will be built.
Is there an integration test that sets tracing to 100% and does not mock out the rest?

Reply:
No, I don't think such a integration test framework exist in istio-proxy repo right? My understanding is that all istio-proxy related integration test needs to be written in istio/istio repo. So we need to add a test there with SHA update.

Reply:
Take a look at test/integration/istio_http_integration_test.cc